src/test/java/uk/gov/pay/adminusers/fixtures/EventFixtureUtil.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.pay.adminusers.fixtures;
+
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.SendMessageResult;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.apache.commons.lang3.NotImplementedException;
+import uk.gov.pay.adminusers.infra.SqsTestDocker;
+
+public class EventFixtureUtil {
+
+    public static String insert(AmazonSQS sqsClient, String eventType, String serviceId, Boolean live, String resourceExternalId,
+                                String parentResourceExternalId, String eventData) {
+        String messageBody = String.format("{" +
+                        "\"resource_external_id\": \"%s\"," +
+                        "\"service_id\": \"%s\"," +
+                        "\"live\": \"%s\"," +
+                        (parentResourceExternalId == null || parentResourceExternalId.isEmpty() ? "%s" : "\"parent_resource_external_id\": \"%s\",") +
+                        "\"event_type\":\"%s\"," +
+                        "\"event_details\": %s" +
+                        "}",
+                resourceExternalId,
+                serviceId,
+                live,
+                parentResourceExternalId == null ? "" : parentResourceExternalId,
+                eventType,
+                eventData
+        );
+
+        SendMessageResult result = sqsClient.sendMessage(SqsTestDocker.getQueueUrl("event-queue"), messageBody);
+        return result.getMessageId();
+    }
+
+    public static PactDslJsonBody getAsPact(String serviceId, Boolean live, String eventType, String resourceExternalId,
+                                            String parentResourceExternalId, JsonNode eventData) {
+        PactDslJsonBody message = new PactDslJsonBody();
+
+        message.stringType("event_type", eventType);
+        message.stringType("resource_external_id", resourceExternalId);
+        message.booleanType("live", live);
+        if (parentResourceExternalId != null && !parentResourceExternalId.isEmpty()) {
```

Comment:
Think it'll be nicer to use `StringUtils.isBlank` here or pass in an `Optional<String> parentResourceExternalId` in the method.

Reply:
Sure, I just copied this bit from ledger but agree so happy to make that change!